researcher_ids/RI_Claims_V1.md

```diff
@@ -89,21 +89,21 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 -   A GA4GH-compatible access token ("GA4GH Access Token"), as per the [GA4GH
     AAI specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md),
-    along with the [Passport JWT Claim](#passport-jwt-claim) that is
+    along with the [Passport Claim](#passport-claim) that is
     returned from [Passport Broker](#passport-broker) service endpoints
     using such an access token.
 
-#### Passport JWT Claim
+#### Passport Claim
 
--   The `ga4gh_passport_v1` JWT claim. It is a [GA4GH
+-   The `ga4gh_passport_v1` claim returned in the response of UserInfo or other OIDC provider endpoints. It is a [GA4GH
```

Comment:
Mikael is looking to separate out endpoints, tokens, and stuff further from the passport spec and not add to it. Therefore it is better to refer to this as a GA4GH Claim only and let that part of the spec explain it.

Reply:
OK, I have removed that.